synthtool/gcp/common.py

```diff
@@ -94,7 +94,7 @@ def node_library(self, **kwargs) -> Path:
         kwargs["publish_token"] = node.get_publish_token(kwargs["metadata"]["name"])
 
         # generate root-level `src/index.ts` to export multiple versions and its default clients
-        if kwargs["versions"] and kwargs["default_version"]:
+        if "versions" in kwargs and "default_version" in kwargs:
```

Comment:
echoing @alexander-fenster's comment, I'm not the best Python developer these days. This looks good to me, but could we add a test to confirm the behavior perhaps?